client/src/Movies/SavedList.js

```diff
@@ -10,9 +12,13 @@ export default class SavedList extends Component {
       <div className="saved-list">
         <h3>Saved Movies:</h3>
         {this.props.list.map(movie => (
+
+          <NavLink to ={`/movies/${movie.id}`} key={movie.id} activeClassName="saved-active">
           <span className="saved-movie">{movie.title}</span>
+          </NavLink>
         ))}
-        <div className="home-button">Home</div>
+        <Link to="/"><div className="home-button">Home</div></Link>
+      
       </div>
     );
```

Comment:
Line 20 - you could also use a `NavLink` for the home button so that you can indicate to the user if they are currently in the home page.